prow/manifests/overlays/metal3/config.yaml

```diff
@@ -135,6 +135,22 @@ branch-protection:
                 contexts:
                   [
                     "test-centos-e2e-integration-main",
+                    "metal3-bmo-e2e-test-pull"
```

Comment:
I think the linter is complaining about this since it is a prow job. There is no need to have it in the required contexts when it is set as a postsubmit further down

Reply:
Yeah, read the comment that this list is only for prow-external tests, which bmo e2e is not. Removed.